### PR TITLE
[EGD-7877] Call errors while MTP file transfer

### PR DESCRIPTION
**Pull request by niepiekm**
Enabled USB device task to handle USB requests outside IRQ handler

### CODE REVIEW

File: composite.c [hunk 1]
```diff
@@ -314,6 +331,20 @@ usb_device_composite_struct_t* composite_init(userCbFunc callback, void* userArg
...
+                    3072L / sizeof(portSTACK_TYPE),  /* task stack size */
+                    composite.deviceHandle,          /* optional task startup argument */
+                    tskIDLE_PRIORITY,                /* initial priority */
+                    &composite.device_task_handle    /* optional task handle to create */
+                    ) != pdPASS)
+    {
+        LOG_ERROR("[Composite] Failed to create usb device task!");
+        return NULL;
+    }
+    #endif
```
Comment (wojtekrzepecki) [inline composite.c:346]: Please do not run threads which are doing nothing while USB not connected. It add substantial CPU load which is especially important for Bell.

Comment (niepiekm) [reply composite.c:346]: You're right, this has to be refactored further. This is a kind of first quick fix to the issue.

Comment (niepiekm) [reply composite.c:346]: Actually, this task is needed as is. In principle, this task sleeps most of the time on OSA_MsgQGet() to get a message put by USB_OTG1 IRQ handler.

Comment (wojtekrzepecki) [reply composite.c:346]: It will be great if you could measure if any cpu overhead occurs.

**Review by wojtekrzepecki — APPROVED** [review]

**Review by pholat — APPROVED** [review]

**Pull request merged**